archive-press.php

```diff
@@ -0,0 +1,326 @@
+      $date = $highlight_meta['_igv_press_date'][0];
+      $author = $highlight_meta['_igv_press_author'][0];
+      $link = $highlight_meta['_igv_press_url'][0];
+
+      if ($highlight_post == 0) { // Image first
+?>
+        <div class="col col-l-12 flex-row align-center">
+          <div <?php post_class('col col-l col-l-6'); ?> id="post-<?php the_ID(); ?>">
+            <a href="<?php echo esc_url($link); ?>" target="_blank">
+              <?php the_post_thumbnail('col-6'); ?>
+            </a>
+          </div>
+          <div <?php post_class('col col-l col-l-6'); ?> id="post-<?php the_ID(); ?>">
+            <a href="<?php echo esc_url($link); ?>" target="_blank">
+<?php 
+        if (!empty($publication)) {
+?>
+              <div class="font-size-h3 margin-bottom-micro"><?php echo $publication; ?></div>
+<?php 
+        }
+?>
+              <h3 class="font-size-h2 margin-bottom-micro">"<?php the_title(); ?>"</h3>
+<?php 
+        if (!empty($author)) {
+?>
+              <div class="font-size-h4"><?php _e('[:en]by[:es]por'); echo ' ' . $author; ?></div>
+<?php 
+        }
+?>
+            </a>
+          </div>
+        </div>
+<?php
+      } else { // Text first
+?>
+        <div class="col col-l-12 flex-row align-center">
+          <div <?php post_class('col col-l col-l-6'); ?> id="post-<?php the_ID(); ?>">
+            <a href="<?php echo esc_url($link); ?>" target="_blank">
+<?php 
+        if (!empty($publication)) {
+?>
+              <div class="font-size-h3 margin-bottom-micro"><?php echo $publication; ?></div>
+<?php 
+        }
+?>
+              <h3 class="font-size-h2 margin-bottom-micro">"<?php the_title(); ?>"</h3>
+<?php 
+        if (!empty($author) || !empty($date)) {
+?>
+              <div class="font-size-h4">
+                <?php 
+                  if (!empty($author)) {
+                    _e('[:en]by[:es]por'); 
+                    echo ' ' . $author;
+                  }
+                ?> 
+                <?php echo (!empty($date) && !empty($author) ? ' | ' : '' ); ?>
+                <?php _e(!empty($date) ? date('j F Y', $date) : '' ); ?>
+              </div>
+<?php 
+        }
+?>
+            </a>
+          </div>
+          <div <?php post_class('col col-l col-l-6'); ?> id="post-<?php the_ID(); ?>">
+            <a href="<?php echo esc_url($link); ?>" target="_blank">
+              <?php the_post_thumbnail('col-6'); ?>
+            </a>
+          </div>
+        </div>
+<?php 
+      }
+      $highlight_post++;
+    }
+?>
+      </div>
+    </div>
+  </section>
+<?php 
+  }
+
+  wp_reset_postdata();
+?>
+
+<?php
+if( have_posts() ) {
+?>
+  <section id="press-posts" class="section">
+    <div class="container">
+      <div class="row">
+        <div class="col col-l col-l-12">
+          <h2><?php _e('[:en]Selected Coverage[:es]Prensa seleccionada'); ?></h2>
+        </div>
+      </div>
+      <div class="row">
+<?php 
+  while( have_posts() ) {
```

Comment:
why to `while`s have no space after function name and spaces inside the args, but `if`s have no space inside args and a space after function name